.github/workflows/stylelinter.yml

```
@@ -0,0 +1,29 @@
+name: stylelinter
```

```suggestion
name: Stylelint
```
Don't want to nitpick, but can we uppercase the check name so it fits in with the others? :slightly_smiling_face: